brats/transformations.py

```diff
@@ -73,3 +76,87 @@ def hist_match(source: np.ndarray, template: np.ndarray) -> np.ndarray:
                                     template_values)
         result[positive] = interp_t_values[bin_idx]
         return result
+
+
+class NiftiOrderTransformation:
+    """
+    Changes dimensions order from nifti (H,W,D,C) to torch convention (C,D,W,H).
+    It can be applied to both ``torch.Tensor`` or ``numpy.ndarray``.
+    """
+
+    def __call__(self, img):
+        if type(img) not in [np.ndarray, torch.Tensor]:
```

Comment:
Tuple can be passed to second argument of is `instanceof` so you can (and probably should) use  `isinstance(img, (np.ndarray, torch.Tensor))`.

Reply:
Not relevant anymore